src/components/Search/utils.ts

```diff
@@ -15,47 +14,8 @@ function belongsToState(county: County, stateFips: string | null) {
   return county.state.fipsCode === stateFips;
 }
 
-export function getSearchAutocompleteLocations(region?: Region) {
-  const allStates = regions.states;
-  const allCounties = regions.counties;
-  const allmetros = regions.metroAreas;
-
-  const sortedStates = sortBy(allStates, state => state.name);
-  const sortedCounties = sortBy(allCounties, county => county.name);
-  const sortedMetros = sortBy(allmetros, metro => metro.name);
-
-  // Homepage:
-  if (!region) {
-    return [...sortedStates, ...sortedMetros, ...sortedCounties];
-  }
-
-  // TODO (chelsi): use same logic as state/county pages? (return counties in metro first, and then?)
-  if (region instanceof MetroArea) {
-    const [countiesInMetro, otherCounties] = partition(sortedCounties, county =>
-      region.counties.includes(county),
-    );
-    return [
-      ...countiesInMetro,
-      ...sortedStates,
-      ...otherCounties,
-      ...sortedMetros,
-    ];
-  }
-
-  if (region instanceof State || region instanceof County) {
-    const stateFips = getStateFips(region);
-    const [stateCounties, otherCounties] = partition(sortedCounties, county =>
-      belongsToState(county, stateFips),
-    );
-    return [
-      ...stateCounties,
-      ...sortedStates,
-      ...sortedMetros,
-      ...otherCounties,
-    ];
-  } else {
-    return [];
-  }
+export function getSearchAutocompleteLocations() {
```

Comment:
Won't this break the behavior where it shows specific locations for the current state? Im not sure we want to be doing that?